ci-operator/config/stackrox/stackrox/stackrox-stackrox-nightlies__ocp-4-13.yaml

```diff
@@ -21,68 +21,97 @@ resources:
 test_binary_build_commands: .openshift-ci/dispatch.sh test-binary-build-commands
 tests:
 - as: qa-e2e-tests
-  postsubmit: true
```

Comment:
These need to stay as postsubmit.

Reply:
Needed to have it as presubmit for rehearsing. Now reverted.